src/Formats/ProtobufSerializer.cpp

```diff
@@ -3508,7 +3513,7 @@ namespace
                 }
 
                 default:
-                    throw Exception(ErrorCodes::LOGICAL_ERROR, "Unknown data type: {}", data_type->getName());
+                    throw Exception(ErrorCodes::ILLEGAL_COLUMN, "Type {} is not supported in Protobuf format", data_type->getName());
```

Comment:
Let's not throw `LOGICAL_ERROR` here. We can add new types in future and forget to support it in Protobuf format. It's ok to thow just `ILLEGAL_COLUMN`.